src/angular/src/app/spark-core-angular/components/sprk-masthead/sprk-masthead.component.ts

```diff
@@ -0,0 +1,108 @@
+import { Component, Input, HostListener } from '@angular/core';
+
+@Component({
+  selector: 'sprk-masthead',
+  template: `
+    <header [ngClass]="getClasses()" role="banner">
+      <div class="sprk-c-Masthead__content">
+        <div class="sprk-c-Masthead__top-row">
+          <div class="sprk-c-Masthead__hamburger">
+            <button class="sprk-c-Hamburger" type="button" [attr.aria-expanded]="isNarrowNavOpen ? true : false"
```

Comment:
Some apps may want the choice to have a different menu icon here.

Reply:
gonna get some feedback from @yodasw16  / @philrinke on this, not sure if its something we want to allow configuration on

Reply:
I don't think we want apps to change the icon used for the menu. That's a pattern we want to remain consistent. Did you have a use case @afebbraro?

Reply:
@yodasw16  - I could see that we wouldn't want apps to change the icon if you are solely thinking of our particular group. But, I'm thinking of these components in terms of other companies. Some may prefer a different type of menu icon.